apps/block_scout_web/lib/block_scout_web/csp_header.ex

```diff
@@ -19,7 +19,7 @@ defmodule BlockScoutWeb.CSPHeader do
 
     Controller.put_secure_browser_headers(conn, %{
       "content-security-policy" => "\
-        connect-src 'self' #{json_rpc_url} #{config[:mixpanel_url]} #{config[:amplitude_url]} #{websocket_endpoints(conn)} #{czilladx_url} #{trustwallet_url} #{walletconnect_urls};\
+        connect-src 'self' http://localhost:8888 https://testnet-1-xchain.explorer.omni.network https://staging-xchain.explorer.omni.network #{json_rpc_url} #{config[:mixpanel_url]} #{config[:amplitude_url]} #{websocket_endpoints(conn)} #{czilladx_url} #{trustwallet_url} #{walletconnect_urls};\
```

Comment:
should this be parametrized as the rest?

Reply:
didn't find any need for that, these are static values anyway